pages.it/common/git-imerge.md

```diff
@@ -0,0 +1,29 @@
+# git-imerge
+
+> Esegui un'unione (merge) o rebase tra due rami git in modo incrementale.
+> Eventuali conflitti tra i due rami sono tracciati in coppie di commit distinti, per semplificarne la risoluzione.
+> Maggiori informazioni: <https://github.com/mhagger/git-imerge>.
+
+- Avvia un rebase usando imerge (dopo aver fatto il checkout del ramo di lavoro):
+
+`git imerge rebase {{ramo_su_cui_eseguire_il_rebase}}`
+
+- Avvia un'unione usando imerge (dopo aver fatto il checkout dal ramo di lavoro):
+
+`git imerge merge {{ramo_su_cui_eseguire_l_unione}}`
```

Comment:
```suggestion
`git imerge merge {{ramo_da_unire}}`
```
Clearer user token name. The main goal here is to make the user understand the difference between a branch specified using `rebase` (the one to rebase *onto*) and a branch specified using `merge` (the one to merge with the current).